### DISCUSSION TITLE
(void-function static-if)

### DISCUSSION THREAD

**Discussion by sjLambda** [minad/vertico#517] [Q&A]
'Error in post-command-hook (vertico--exhibit): (void-function static-if)
Error in pre-command-hook (vertico--prepare): (void-function static-if)
apply: Command attempted to use minibuffer while in minibuffer
Quit
Error in post-command-hook (vertico--exhibit): (void-function static-if)
Quit
`

I just started getting these error *Messages*. Is this something you're aware? Or something at my end? Where should I look? I'm running:

;; Version: 1.9
;; Package-Requires: ((emacs "27.1") (compat "30"))

on emacs-pgtk compiled from master trunk sources from savannah on NixOS.

**Comment by minad** [comment]
See https://github.com/minad/vertico/discussions/501
